lib/v2/apache/apisix/blog.js

```diff
@@ -0,0 +1,34 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+const timezone = require('@/utils/timezone');
+
+async function getArticles() {
+    const url = 'https://apisix.apache.org/zh/blog/';
+    const { data: res } = await got(url);
+    const $ = cheerio.load(res);
+    const articles = $('section.sec_gjjg').eq(1).find('article');
+    const results = [];
+    articles.each((i, elem) => {
+        const a = $(elem).find('header > a');
+        const time = $(elem).find('footer').find('time').text();
+        const author = $(elem).find('footer').find('img').attr('src');
+        results.push({
+            title: a.find('h2').text(),
+            description: a.find('p').text(),
+            link: a.attr('href'),
+            pubDate: timezone(parseDate(time, 'YYYY年M月D日'), +8),
+            author,
+        });
+    });
```

Comment:
Please use [map](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Array/map) to create a new array.